From 2010074e1daa14518dd9171af842bf72a8eed75b Mon Sep 17 00:00:00 2001
From: zhangqianqian1 <zhangqianqian1@eswincomputing.com>
Date: Wed, 24 Sep 2025 17:01:00 +0800
Subject: [PATCH 522/540] feat:Remove usb_test and kernel dynamic log.

Changelogs:
	1.Remove usb test driver probe.
        2.Close kernel dynamic debug log.

Signed-off-by: zhangqianqian <zhangqianqian1@eswincomputing.com>
---
 arch/riscv/configs/fml13v03_defconfig | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/riscv/configs/fml13v03_defconfig b/arch/riscv/configs/fml13v03_defconfig
index 8d98e892bbfb..f0428afebddd 100755
--- a/arch/riscv/configs/fml13v03_defconfig
+++ b/arch/riscv/configs/fml13v03_defconfig
@@ -4393,8 +4393,8 @@ CONFIG_PRINTK_TIME=y
 CONFIG_CONSOLE_LOGLEVEL_DEFAULT=15
 CONFIG_CONSOLE_LOGLEVEL_QUIET=15
 CONFIG_MESSAGE_LOGLEVEL_DEFAULT=7
-CONFIG_DYNAMIC_DEBUG=y
-CONFIG_DYNAMIC_DEBUG_CORE=y
+CONFIG_DYNAMIC_DEBUG=n
+CONFIG_DYNAMIC_DEBUG_CORE=n
 CONFIG_VMLINUX_MAP=y
 CONFIG_MAGIC_SYSRQ=y
 CONFIG_DEBUG_FS=y
-- 
2.54.0

